keps/sig-scheduling/1923-prefer-nominated-node/README.md

```diff
@@ -146,10 +143,170 @@ _This section must be completed when targeting alpha to a release._
     - Feature gate name: PreferNominatedNode
     - Components depending on the feature gate: kube-scheduler
 
+* **Does enabling the feature change any default behavior?**
+  Yes. The pod with the nominated node has been set will be evaluated first in any scheduling cycle,
+  this is only the default process logic that is handled by scheduler, end-user will not
+  and need not aware of any difference.
+
+* **Can the feature be disabled once it has been enabled (i.e. can we roll back
+  the enablement)?**
+  We can roll back the enablement by restarting scheduler with the feature gate turned off, otherwise
```

Comment:
The question is more whether it is possible (i.e. if something will not break when you rollback because some state in k8s etc.). Here it's purely runtime in-memory feature so the answer is no.

So let's change to something like:

Yes - it can be disabled by restarting scheduler with feature gate turned off.

Reply:
Great suggestion, will update soon.